### DISCUSSION TITLE
Prisma update where clause is too strict

### DISCUSSION THREAD

**Discussion by frankiesardo** [prisma/prisma#7167] [Q&A] [answered]
Heya. I just saw that the syntax to update a single row has a very restrictive where clause https://www.prisma.io/docs/concepts/components/prisma-client/crud#update

Basically, if I want to update an `Article`, I can only do so via `db.article.update({ where: { id } , data })`
I want to be able to add more conditions in the where clause. Right now, I want to be able to say `update({ where: { id, authorId: session.userId }`

The alternative is to do another query first to check if the current logged in user is the author, which is not too bad, but an ORM should be able to support complex update logic. Any reason why it's not allowed in Prisma?

**Comment by ryands17** [comment, marked as answer]
@frankiesardo 👋 

`update` is actually for a single record which is why it only accepts unique fields. For adding multiple conditions, you can use [`updateMany`](https://www.prisma.io/docs/concepts/components/prisma-client/crud#update-multiple-records) that will accept all the fields.
